common_v2.proto

```diff
@@ -0,0 +1,164 @@
+syntax = "proto3";
+option optimize_for = SPEED;
+option java_multiple_files = true;
+option go_package = "fivetran.com/fivetran_sdk";
```

Comment:
we need add the version number to the go package too. I think it would be `"fivetran.com/fivetran_sdk/v2"`